protostar/testing/test_config.py

```diff
@@ -1,6 +1,6 @@
 from dataclasses import dataclass, field
 from enum import Enum
-from typing import Dict
+from typing import Dict, List
```

Comment:
[Consistency] We started using Python 3.9 types: `dict`, `list`.

Reply:
[use built-in types for dict and tuple](https://github.com/software-mansion/protostar/pull/947/commits/8c50898fc9488104046d067b20440d3335dfd6bc)